sdks/go/pkg/beam/core/graph/fn.go

```diff
@@ -104,6 +106,14 @@ func NewFn(fn interface{}) (*Fn, error) {
 
 	case reflect.Struct:
 		methods := make(map[string]*funcx.Fn)
+		annotations := make(map[string][]byte)
+		af := val.Indirect().FieldByName("Annotations")
```

Comment:
Indirect is a package function, not a method on Value. You'll need to change this up with
```suggestion
		af := reflect.Indirect(val).FieldByName("Annotations")
```
https://golang.org/pkg/reflect/#Indirect